axon-server-connector/src/test/java/org/axonframework/axonserver/connector/heartbeat/FakeClock.java

```diff
@@ -0,0 +1,43 @@
+package org.axonframework.axonserver.connector.heartbeat;
+
+import java.time.Clock;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.util.function.Supplier;
+
+/**
```

Comment:
How ever small, I think this block would benefit from a one-line description what this class is for.